main.go

```diff
@@ -78,6 +82,11 @@ func newOutdatedCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 		Aliases: []string{"od"},
 		Args:    require.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
+                        if showVersion {
+		                fmt.Fprintln(out, "Version:", version)
+		                fmt.Fprintln(out, "GitCommit:", gitCommit)
+                                os.Exit(0);
```

Comment:
Instead of manually exiting, you should simply `return nil`.
```suggestion
		                return nil
```